resources/analysisTools/qcPipeline/qcJson.pl

```diff
@@ -218,15 +218,21 @@ ()
 	my $exp1 = {
           '1' => {
                    'genomeWithoutNCoverageQcBases' => '0.01',
-                   'chromosome' => '1',
+			       'genomeWithoutNReferenceLength' => '225280621',
```

Comment:
What are these numbers? Can they be configured? Where do they come frome? Are they really always the same?

Reply:
Yes. Its the numbers I put into the test above. It's simply testing whether I parse out the correct numbers.